website/docs/reference/glossary.md

```diff
@@ -0,0 +1,140 @@
+---
+sidebar_position: 2
+---
+
+# Glossary
+
+## List of FerretDB terminologies
+
+*This section contains a list of common terminologies related to FerretDB*.
+
+### A
+
+#### aggregation
+
+A group of operations used to reduce or merge large data sets.
```

Comment:
In principle, data sets don't have to be large. It can be executed with small data sets too. The idea is that during aggregation, we can execute some special operations - like compute some statistics, or some math operators etc. Aggregation can be split into a few stages, e.g. first we filter the necessary data and then execute some statistics calculation for this data. But such verbal description is not for the glossary I guess :)

So, overall, I'd say the idea of aggregation is that we process documents and apply various operations to them.